src/components/HomeLayout/Helpline.js

```diff
@@ -26,30 +82,74 @@ const HelplineModal = ({ showModal, setShowModal }) => (
           <a href={helplineLinks.english} target="_new">
             <Button type="primary">English</Button>
           </a>
-        </span>
+        </span> */}
       </div>
-    </div>
-  </Modal>
+    </div >
+  </Modal >
 );
 
+// const HelplineModal = ({ showModal, setShowModal }) => (
+//   <Modal
+//     title={<div>Women&apos;s Helpline</div>}
+//     visible={showModal}
+//     onOk={() => setShowModal(false)}
+//     onCancel={() => setShowModal(false)}
+//     footer={null}
+//   >
+//     <div style={{ textAlign: "left" }}>
+//       <div className="modal-txt">
+//         <p> Helpline numbers are available in the links below, </p>
+//         <span>
+//           <a href={helplineLinks.kannada} target="_new">
+//             <Button type="primary">ಕನ್ನಡ</Button>
+//           </a>
+//         </span>
+//         <Spacer width={10} />
+//         <span>
+//           <a href={helplineLinks.english} target="_new">
+//             <Button type="primary">English</Button>
+//           </a>
+//         </span>
+//       </div>
+//     </div >
+//   </Modal >
+// );
+
 const Helpline = () => {
   const [showModal, setShowModal] = useState(false);
 
   return (
+    // <Row>
+    //   <HelplineModal showModal={showModal} setShowModal={setShowModal} />
+
+    //   <Col span={24} className="bottom-btn-containers-alt">
+    //     <div
+    //       className="goto-text ftl"
+    //       style={{ textAlign: "center" }}
+    //       onClick={() => setShowModal(true)}
+    //     >
+    //       <Button className="orange-btn">
+    //         <WomanOutlined /> Women&apos;s Helpline
+    //       </Button>
+    //     </div>
+    //   </Col>
+    // </Row>
+    // <Row>
     <Row>
-      <HelplineModal showModal={showModal} setShowModal={setShowModal} />
+      <TravelAssistanceModal showModal={showModal} setShowModal={setShowModal} />
+
       <Col span={24} className="bottom-btn-containers-alt">
         <div
           className="goto-text ftl"
           style={{ textAlign: "center" }}
           onClick={() => setShowModal(true)}
         >
           <Button className="orange-btn">
-            <WomanOutlined /> Women&apos;s Helpline
-          </Button>
+            Travel Assistance
```

Comment:
You can use the <CarOutlined /> icon from ant design to be consistent with the previous link and behaviour

Reply:
`<CarOutlined />` icon

Reply:
Sure will add the icon